internal/pkg/config/exclusions.go

```diff
@@ -0,0 +1,151 @@
+package config
+
+import (
+	"bufio"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+)
+
+func (c *Config) exclusionFileLiveReloader() {
+	defer config.waitGroup.Done()
```

Comment:
looks like don't need to call the Done() after switching to the new `waitgroup.Go()` method.

Reply:
Duh! Thx